jcl/src/java.base/share/classes/java/lang/invoke/FieldGetterHandle.java

```diff
@@ -24,6 +24,11 @@
 
 import java.lang.reflect.Field;
 import java.lang.reflect.Modifier;
+/*[IF Sidecar19-SE]
+import jdk.internal.misc.Unsafe;
+/*[ELSE]*/
+import sun.misc.Unsafe;
+/*[ENDIF]*/
```

Comment:
Are these necessary? As long as you don't mention the Unsafe type, you shouldn't need the imports.

Reply:
No, updating it now.